lib/block-editor-settings.php

```diff
@@ -53,6 +53,13 @@ function gutenberg_get_block_editor_settings( $settings ) {
 			$global_styles[]      = $block_classes;
 		}
 
+		// Get any additional css from the customizer and add it before global styles custom CSS.
```

Comment:
```suggestion
		// Get any additional css from the customizer and add it after global styles custom CSS.
```

From what I've tested, the Customizer CSS seems to be loaded _after_ the global styles 🤔

```html
<iframe src="http://localhost:8888" title="front-end view" style="display: block; width: 100%; height: 100%;">
<html lang="en-US" style="margin-top: 0 !important;">

<head>
<style id="global-styles-inline-css" type="text/css"></style>
<style type="text/css" id="wp-custom-css"></style>
</head>

<body>
</body>

</html>
</iframe>
```

Reply:
It's loaded after global styles, but before global styles custom CSS - so anything that you put in the "Additional CSS" field can override any custom styles from the customizer. It's unlikely that you'll find both in one website? But at some point it was thought that global styles custom CSS should take precedence over customizer custom CSS.